pkg/commands/build/status.go

```diff
@@ -75,17 +67,12 @@ Therefore, you must have credentials to access the registry on your machine when
 					return err
 				}
 
-				if bom {
-					return displayBOM(authn.DefaultKeychain, cmd, bld, rup, tlsConfig)
-				} else {
-					return displayBuildStatus(cmd, bld)
-				}
+				return displayBuildStatus(cmd, bld)
 			}
 		},
 	}
 	cmd.Flags().StringVarP(&namespace, "namespace", "n", "", "kubernetes namespace")
 	cmd.Flags().StringVarP(&buildNumber, "build", "b", "", "build number")
-	cmd.Flags().BoolVar(&bom, "bom", false, "only print the built image bill of materials")
 	commands.SetTLSFlags(cmd, &tlsConfig)
```

Comment:
these flags can be removed too! no more tls needed!

Reply:
Thanks for your comment! Please see https://github.com/vmware-tanzu/kpack-cli/pull/261/commits/7d5a6a995a31b5d41fb264c0a9ef39c831b40540.